test/unit/mdc-animation/mdc-animation.test.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import test from 'tape';
+import td from 'testdouble';
+
+import {getAnimationEventName} from '../../../packages/mdc-animation';
+
+// Has no properties without a prefix
+const legacyWindowObj = td.object({
+  document: {
+    createElement: (str) => {
+      return {
+        style: {
+          'webkitTransform': {},
+        },
+      };
+    },
+  },
+});
+
+test('#getAnimationEventName does not prefix events and properties when not necessary', (t) => {
+  const windowObj = window;
```

Comment:
I'd honestly still mock out window here. Doing so will make these tests deterministic such that you'll always get the same result in any browser. For example, this test may fail on older versions of Safari.